packages/uhk-web/src/app/components/popover/tab/keypress/keypress-tab.component.ts

```diff
@@ -104,6 +104,9 @@ export class KeypressTabComponent extends Tab implements OnChanges {
         if (keystrokeAction.secondaryRoleAction !== undefined) {
             this.selectedSecondaryRoleIndex = this.mapper.modifierMapper(keystrokeAction.secondaryRoleAction);
         }
+        else {
```

Comment:
This seems to work well, but would you please write `} else {` in the same line according to our coding standards?

Reply:
done